packages/desktop-client/src/components/mobile/transactions/TransactionListItem.tsx

```diff
@@ -71,8 +71,9 @@ const getScheduleIconStyle = ({ isPreview }: { isPreview: boolean }) => ({
   color: isPreview ? theme.pageTextLight : theme.menuItemText,
 });
 
-type TransactionListItemProps = ComponentPropsWithoutRef<
-  typeof ListBoxItem<TransactionEntity>
+type TransactionListItemProps = Omit<
+  ComponentPropsWithoutRef<typeof ListBoxItem<TransactionEntity>>,
+  'onPress' | 'onLongPress'
```

Comment:
ListBoxItem does not have onLongPress so no need to omit this
```suggestion
  'onPress'
```